File: core/src/main/java/com/sys1yagi/swipe/core/entity/Item.java
package com.sys1yagi.swipe.core.entity;

import com.google.gson.annotations.SerializedName;

public class Item {

    @SerializedName("url")
    String url;

    @SerializedName("title")
    String title;

    @SerializedName("icon")
    String icon;

    public String getUrl() {
        return url;
    }

    public String getTitle() {
        return title;
    }

    public String getIcon() {
        return icon;
    }
}

File: core/src/main/java/com/sys1yagi/swipe/core/entity/index/Item.kt
package com.sys1yagi.swipe.core.entity.index

import com.google.gson.annotations.SerializedName

class Item {

    @SerializedName("url")
    var url: String? = null

    @SerializedName("title")
    var title: String? = null

    @SerializedName("icon")
    var icon: String? = null

}
